test/spec/NodeConnection-test.js

```diff
@@ -192,6 +192,42 @@ define(function (require, exports, module) {
             });
         });
         
+        it("should receive progress events from asynchronous commands", function () {
```

Comment:
@abose Have you taken shell changes also? https://github.com/adobe/brackets-shell/pull/509